v2/custom_resources/alert_sample_app/json/resource_types/alert.json

```diff
@@ -0,0 +1,18 @@
+{
+  "data": {
+    "key": "alert",
+    "schema": {
+      "properties": {
+        "contents": {
+          "type": "string",
+          "description": "The contents of the alert"
+        },
+        "level": {
+          "type": "string",
```

Comment:
Is there an enum concept in JSONAPI, or validation of the value to be one of the things listed below?

Reply:
Yes there is an enum